tests/gdx-tests/src/com/badlogic/gdx/tests/utils/GdxTests.java

```diff
@@ -209,6 +209,7 @@
 		SpriteBatchTest.class,
 		SpriteCacheOffsetTest.class,
 		SpriteCacheTest.class,
+		SpriteExtraAttributesTest.class,
```

Comment:
Please also add this test to the GwtTestWrapper, see https://github.com/libgdx/libgdx/wiki/Running-Tests#creating-new-tests